kronolith/lib/Event.php

```diff
@@ -775,7 +782,9 @@ public function toiCalendar($calendar)
         $vEvent->setAttribute('SUMMARY', $this->getTitle());
 
         // Organizer
-        if (count($this->attendees)) {
+        if ($this->organizer) {
+            $vEvent->setAttribute('ORGANIZER', 'mailto:' . $this->organizer, array());
+        } elseif (count($this->attendees)) {
```

Comment:
Only one Attendee is organizer. Not sure if he needs to attend at all. This means use ORGANIZER if available, fall back to CREATOR if not, have no ORGANIZER if it has no attendees (is an event but no appointment).